src/platform/user/profile/vap-svc/util/local-vapsvc.js

```diff
@@ -6,7 +6,7 @@ export function isVAProfileServiceConfigured() {
     // using the existence of VetsGov.pollTimeout as an indicator that we are
     // running unit tests and therefore _do_ want the FE to make real API calls
     window.VetsGov.pollTimeout ||
-    window.Cypress ||
+    (window.Cypress && !window.specRequiresAddress) ||
```

Comment:
@CBonade I believe that if you properly mocked the `GET user/` call in your Cypress test this addition to the conditional would not be required. Can you please try that and revert this change?

Reply:
https://github.com/department-of-veterans-affairs/vets-website/pull/17656

Thanks for the feedback here!  I was able to rework the user mock as you suggested and get this test passing with a different user structure.  New PR linked above that adds the new user structure and removes the wacky conditional.  Thanks again!